gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -124,9 +130,20 @@ def main(args):
                 "Adding VEP context annotations and preparing tables for constraint"
                 " calculations..."
             )
-            # TODO: Need to add function that annotates methylation, coverage, and
-            #  gerp in the vep context table.
-            context_ht = annotated_context_ht.versions[version].ht()
+            # Annotates methylation, coverage, and gerp in the vep context Table.
+            if prepare_context_ht:
+                split_context_ht(
+                    vep_context.versions["101"].ht(),
```

Comment:
We probably want to make this an argument, or have a dictionary indicating what context to use for each release